src/SFProvider.cc

```diff
@@ -15,11 +17,15 @@ SFProvider::SFProvider(std::string_view input_file, std::string_view channel, st
         ss << "tau_trigger::SFProvider: unable to open \"" << input_file << "\".";
         throw std::runtime_error(ss.str());
     }
-
+    
+    bool isUL = boost::algorithm::contains(input_file,"UL");
+    if(isUL){
+      comb_decay_modes = { 0, 1, 10, 11,1011};
```

Comment:
Again I don't think it should be possible for the user to extract the scale factors and efficiencies for decay modes 10 and 11 separately.